### DISCUSSION TITLE
Recent update seems to have broken app or possibly caching

### DISCUSSION THREAD

**Discussion by GilBatzri** [Ascoware/get-iplayer-automator#475] [Q&A] [answered]
First particulars about configuration:
GIA Version 1.26.0 (20240101001)
OSX 10.15.7 (2012 MBP, i7, 16GB RAM, SSD boot device)

I have successfully been using GIA to download BBC Radio programmes for maybe 5 or 6 years, if my memory serves.

I updated to the 1.26 version when it released, and got some odd errors that broke my install, I was able to get it working after a fashion by rolling back to 125.4 (kind of) 

The Cacheing of programme data seems to have been corrupted, and I was unable to get updates past some time in January, I was for a period able to use safari and pull my episodes into the queue that way. But the update cache did not import new data.

I tried the update again today, and it completed successfully, but the cache doesn't seem to be updating (it updates, but searches all respond no entries found or something similar)  New behavior is that the webbrowser import function no longer works.

I searched for some detail on manually deleting the cache, and found this ancient bit of documentation.
https://code.google.com/archive/p/get-iplayer-automator/wikis/CleanInstall.wiki

I followed that finding the current entries in the libraries, vs the obsoltete listed ones in the doc, and reset the app basically, but it still is non-functional. 

Is there some current documentation on the file locations for the caching, or some way to clear the caches I am unaware of. I haven't been able to download my programmes from the beeb and many are about to scroll off.

Any help would be appreciated, I love this software.

**Comment by skovatch** [comment]
In order to figure out what's going on with caching I need to get more detailed information about the caching progress. I will put a build on my Google Drive that will dump out just about every message from the cache update and search. 

https://drive.google.com/file/d/1vaTn6iYTUDFBCyNZ-rG6zkjCDUN1Nzbf/view?usp=sharing

I would greatly appreciate it if you could launch it, turn on verbose mode, and then save the output from the log window to a file and attach it here. Hopefully I can get some insight as to what's failing, as I'm not having any problems with caching or searching.

**Reply by GilBatzri** [reply]
I'll try now

Thanks!

**Comment by GilBatzri** [comment]
[GIA Log.txt](https://github.com/Ascoware/get-iplayer-automator/files/14798574/GIA.Log.txt)
Hopefully that will work, Textedit will not let me save as .txt, which is funny in a way. So that is from Word.

definitely a bunch of unhapppiness in the log.

**Reply by skovatch** [reply]
This is great, thank you! I'm still a bit stumped as to why it works for me, but I think I have a fix. I'll post a new test build shortly.

**Comment by skovatch** [comment, marked as answer]
Please try: https://drive.google.com/file/d/16eyScLgheD2C7xVEUn9kx810g-iIGb7Q/view?usp=drive_link
and let me know how it goes.

**Comment by GilBatzri** [comment]
YOU ARE AMAZING.

Downloads are working for me, I can't tell you how impressed I am.

**Comment by skovatch** [comment]
Excellent! I will push out a new release with this fix.
